### ISSUE TITLE
Homunculus Experience Percentage are not always Updating

### ISSUE THREAD

**Issue by lucasca73** [OpenKore/openkore#1810]
###### Summary: 
When running "homun s" command on console. The info about experience percentage are not always updating. But current exp and max exp are. This is not a worry, for me i changed the code in Commands.pm to calculate based on current exp and the max exp. The variable may not be updating all the time... dont know.

## Was...

`defined $slave->{exp}
			? T("Exp: ") . formatNumber($slave->{exp})."/".formatNumber($slave->{exp_max})." (".sprintf("%.2f",$slave->{expPercent})."%)"`

## Now for me...
`defined $slave->{exp}
			? T("Exp: ") . formatNumber($slave->{exp})."/".formatNumber($slave->{exp_max})." (".sprintf("%.2f",($slave->{exp}/$slave->{exp_max})*100 )."%)"`

###### Impact:
Minimum

###### Expected Behavior:
Always Update

###### Actual Behavior:
Stuck in old information

###### Steps to Reproduce:
run "homun s" in console when levelling homunculus

**Comment by Nipodemos** [comment]
you can make a pull request, but i don't know if they will approve, since the best option was to fix the $slave->{expPercent} instead of stop using it

**Comment by lucasca73** [comment]
Yes the best way is to correct. I am just warning about this silly bug.

**Comment by lututui** [comment]
In this case, the best solution is in fact to ditch {expPercent}. The same can be said about {hpPercent} and {spPercent}

**Comment by moneyisgod** [comment]
@lucasca73 How about ?!
